### PR TITLE
Add list js

### PR DESCRIPTION
**Pull request by mandelatuks**
- Add js file with different functionality

### CODE REVIEW

File: list.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+document.querySelector('#delete-all').addEventListener('click', () => {
+  todolist.clearAll();
+});
+
+class List {
+  constructor() {
+    this.list = JSON.parse(localStorage.getItem('todolist'));
+    if (!this.list) {
+      this.list = [];
+    }
```
Comment (JauJau088) [inline list.js]: In this case, a Short-circuit Evaluation as explained [here](https://codeburst.io/javascript-short-circuit-evaluation-3709ffda6384) can be used. So, all that code can simply be replaced with this single line of code: `this.list = JSON.parse(localStorage.getItem('todolist')) || [];`

**Comment by JauJau088** [comment]
# Status: Changes Required ♻️ 

Hi, @mandelatuks good job on your work.

You are almost there, however, there are some small changes that need to be made for your code to be more in line with coding best practices.



## To Highlights 🏆 

✅ No errors
✅ Clean and easily readable code
✅ Proper use of class

## Required Changes ♻️ 

🔲 Kindly check inline comments for the required changes

## Notes 📓 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @JauJau088 in your question so I can receive the notification.

**Cheers and happy coding** 🥂 👨‍💻

**Comment by mandelatuks** [comment]
@JauJau088 
can't see the needed changes

File: list.js (hunk 1 above)
Comment (mandelatuks) [reply list.js]: okay,am on it

Comment (mandelatuks) [reply list.js]: kindly check for the updated version now

**Comment by JauJau088** [comment]
# Status: Approved ✔️ 

Hi @mandelatuks,

Conratulations 🎉, your project is now complete!



**There is nothing else to say other than... it's time to merge it!** :shipit:

Cheers 🥂

**Pull request merged**